test/tables.jl

```diff
@@ -182,7 +182,7 @@ end
 
      df2 = DataFrame!(eachrow(df))
      @test df == df2
-     @test !any(((a,b),) -> a === b, zip(eachcol(df), eachcol(df2)))
+     @test all(((a,b),) -> a === b, zip(eachcol(df), eachcol(df2)))
```

Comment:
With this PR, `df2 = DataFrame!(eachrow(df))` does not copy columns any more. Is it an OK change?

Reply:
I would say that it is even better. @nalimilan - are you OK with this?
If someone writes `DataFrame!` an explicit opt-out from copying, if possible, is assumed.

Reply:
Yes, makes sense.